### PR TITLE
cincinnati: install gcc for test compilation

### PR DESCRIPTION
**Pull request by petr-muller**
Should fix the `error: linker cc not found` errors surfaced by in https://github.com/openshift/cincinnati/pull/934

### CODE REVIEW

**Comment by Davoska** [comment]
ci/rehearse/openshift/cincinnati/master/e2e will be failing until https://github.com/openshift/cincinnati/pull/935 is merged, or the base image for the e2e test pod is updated.

This change shouldn't affect the e2e test IMO.

File: ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml [hunk 1]
```diff
@@ -19,6 +19,7 @@ images:
     RUN chgrp root /opt/app-root/src/cincinnati && \
         chmod g+w /opt/app-root/src/cincinnati && \
         dnf install -y jq python38 && \
+        dnf install -y gcc && \
```
Comment (Davoska) [inline ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml:22]: nit: 
```suggestion
        dnf install -y jq python38 gcc && \
```

Comment (petr-muller) [reply ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml:22]: :facepalm: 

of course

**Pull request merged**